File: plugins/wise-dev-std/templates/scaffold/react-native-app/src/core/env.ts
// {{PROJECT_NAME}} — flavor → API base (EAS profile / app.json extra 로 주입)
import Constants from "expo-constants";

const extra = (Constants.expoConfig?.extra ?? {}) as Record<string, string>;

export const Env = {
  appEnv: extra.appEnv ?? process.env.EXPO_PUBLIC_APP_ENV ?? "local",
  flavor: extra.flavor ?? process.env.EXPO_PUBLIC_FLAVOR ?? "development",
  apiBaseUrl: extra.apiBaseUrl ?? process.env.EXPO_PUBLIC_API_BASE_URL ?? "http://localhost:8000",
};

File: plugins/wise-dev-std/templates/scaffold/flutter-app/lib/core/env.dart
// {{PROJECT_NAME}} — flavor → API base (build-time via --dart-define-from-file=.env.local)
class Env {
  static const appEnv = String.fromEnvironment('APP_ENV', defaultValue: 'local');
  static const flavor = String.fromEnvironment('FLAVOR', defaultValue: 'dev');
  static const apiBaseUrl =
      String.fromEnvironment('API_BASE_URL', defaultValue: 'http://localhost:8000');
}
